.golangci.yml

```diff
@@ -5,7 +5,7 @@ linters:
   enable:
     - gofmt
     - govet
-    - depguard
+    #- depguard # there's a bug in depguard with (or just exacerbated by?) Go 1.19 that causes it to have false positives
```

Comment:
How often is this happening? I saw it once and yes annoying to restart, I would rather leave it on. Because, as we have already shown with t_stats and other things these get forgotten.

Here is the issue in godep that as a potential fix available as of a few days ago:
https://github.com/OpenPeeDeeP/depguard/issues/25

Reply:
In CI, it's not very bad. For actually working on the codebase locally: OMG it happens all the time.

Reply:
In a bit I'll put up a separate PR to turn it back on and bump depguard to that beta release.

Reply:
Maybe I missed it but do we have the other PR that re-enables this or an issue that tracks it?